ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
           - names: [internet]
             preference: alternate
 ```
+
+### WAN Validation
+
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the different BGP Address Families Identifiers (AFI) and Subsequent Address Family Identifiers (SAFI) sessions state. |
+| AvdTestIPSecurity | VerifySpecificIPSecConn | Validates the establishment of IP security connections for a peer within the default VRF. In its current state, the test validates only IPsec connections defined as static peers under the `router path-selection` section of the configuration. |
+| AvdTestStun | VerifyStunClient | Validates the presence of a STUN client translation for a given source IPv4 address and port for WAN scenarios. The list of expected translations for each device is built by searching local interfaces in each path-group. |
```

Comment:
is there a plan to leverage the AVT tests at some point? https://anta.arista.com/stable/api/tests.avt/

Reply:
Yes in next set of tasks will add AVT tests and DPS reachability test.

Reply:
Thanks! Let's add a comment under the table that more tests are coming